src/script/engines/lua.c

```diff
@@ -5,6 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 #include <mgba/internal/script/lua.h>
 
+#include <mgba/core/core.h>
```

Comment:
What uses `core/core.h`?

Reply:
Nothing, anymore, because I decided to hook the frame callback in Lua code instead of in C code. I forgot about the include.